bootable-jar/WFLY-13305_Bootable_Jar.adoc

```diff
@@ -0,0 +1,450 @@
+
+In the context of Bootable JAR, the script does not need to contain commands to connect to the server or start an embedded server. 
+The Maven plugin handles that for you by starting an embedded server for each group of scripts.
+
+The plugin allows you to execute multiple group of scripts with different CLI contexts. 
+A group of scripts and its configuration are defined in a ```cli-session``` composed of:
+
+* ```<script-files>```: the list of paths to script files .
+* ```properties-file```: (optional) a path to a properties file that contains java properties that scripts can reference (using the syntax ```${my.prop}```). 
+For example, a command that sets the public inet-address to the value of ```all.addresses``` system property looks like: ```/interface=public:write-attribute(name=inet-address,value=${all.addresses})```
+* ```resolve-expressions```: (optional) a boolean indicating if system properties or 
+expressions are resolved before sending the operation requests to the server. Value is ```true``` by default.
+
+All scripts present in a ```cli-session``` are executed within a single CLI execution. An embedded server is started for each defined ```cli-session```.
+
+NB: The scripts are executed in the order they are defined in the plugin configuration. 
+
+CLI configuration example:
+```
+<cli-sessions>
+  <cli-session>
+    <script-files>
+        <script>../scripts/script1.cli</script>
+    </script-files>
+    <!-- We want the env variables to be resolved during server execution -->
+    <resolve-expressions>false</resolve-expressions>
+  </cli-session>
+  <cli-session>
+    <script-files>
+        <script>../scripts/script2.cli</script>
+    </script-files>
+    <properties-file>../scripts/cli.properties</properties-file>
+    <!-- We want the properties to be resolved during CLI execution (not actually needed, this is the default behavior) -->
+    <resolve-expressions>true</resolve-expressions>
+  </cli-session>
+</cli-sessions>
+```
+
+
+===== User application
+
+The WAR or JAR main artifact the Maven project is building is deployed and zipped along with the server. 
+The plugin option root-url-path=true|false (true by default) allows to rename the war file to ROOT.war 
+(doesn’t apply to other packaging types). 
+
+NB: Application deployment doesn't rely on the deployment scanner. The scanner is not required, it is even suggested to exclude it, useless in a bootable JAR context.
+For an example of how to exclude the _deployment-scanner_ Galleon layer, you can check https://github.com/wildfly-extras/wildfly-jar-maven-plugin/blob/master/examples/jaxrs/pom.xml#L35[this example]
+
+In case the bootable is an “hollow JAR”, no deployment is present in the JAR.
+
+===== Hollow JAR
+
+No WAR/JAR/EAR file is copied into the server deployments. This is controlled by a plugin option.
+The hollow JAR doesn’t require the deployment-scanner to be provisioned in order for the deployment to be taken into account at runtime time.
```

Comment:
I'd suggest to s/`at runtime time`/`by the runtime`/

Reply:
Ok

Reply:
Thanks